HolyshiftService/DB models/HolyShiftUser.cs

```diff
@@ -0,0 +1,17 @@
+public class HolyShiftUser
+{
+    public string Id { get; set; }
```

Comment:
Since this is the DBModel be explicit about the primary key by adding attribute [Key].

EFCore uses those attributes to construct the model and migrations.